src/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -629,12 +629,6 @@ public static string Title
 
             set
             {
-                if (value == null)
-                    throw new ArgumentNullException(nameof(value));
-                if (value.Length > MaxConsoleTitleLength)
-                    throw new ArgumentOutOfRangeException(nameof(value), SR.ArgumentOutOfRange_ConsoleTitleTooLong);
-                Contract.EndContractBlock();
-
```

Comment:
It's the same code as in Console.cs: https://github.com/hughbe/corefx/blob/fbca63abe848380b8d07ebfad84ee23a20f2f17d/src/System.Console/src/System/Console.cs#L287-L295